Cargo.toml

```diff
@@ -28,8 +28,7 @@ members = [
 	"pallets/space",
 	"pallets/schema",
 	"pallets/stream",
-	"pallets/authorities",
-	"pallets/builder",
-	"pallets/foundation",
+	"pallets/anchor",
```

Comment:
the word 'anchor' itself is bit confusing when i just check this. (as anchor is still a name we use for 'create' method in many pallets already). Need to document this better.

Reply:
`royalty_treasury`